frontend/stubs/functions.py

```diff
@@ -1,12 +1,20 @@
 """Stub file for built in functions"""
+from typing import TypeVar, List, Tuple, Dict
 
+Any = TypeVar("Any")
```

Comment:
Removed Any and used `T` instead.
AnyStr exists in `typing` but represents the same thing. The rest are fine.